<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWindow</class>
 <widget class="QMainWindow" name="ListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>196</width>
          <height>454</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page1.bmp</normaloff>:/images/page1.bmp</iconset>
        </attribute>
        <attribute name="label">
         <string>第一页</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="m_btnInit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnClear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnAppend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnInsert">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnDelete">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>196</width>
          <height>454</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page2.bmp</normaloff>:/images/page2.bmp</iconset>
        </attribute>
        <attribute name="label">
         <string>第二页</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>196</width>
          <height>454</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page3.bmp</normaloff>:/images/page3.bmp</iconset>
        </attribute>
        <attribute name="label">
         <string>第三页</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page1.bmp</normaloff>:/images/page1.bmp</iconset>
        </attribute>
        <attribute name="title">
         <string> 第一页</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="m_list">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="m_btnCheckAll">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_btnCheckNone">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_btnCheckInverse">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_btnCheck">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="popupMode">
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="m_checkEditable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string> 可编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="m_editChanged">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page2.bmp</normaloff>:/images/page2.bmp</iconset>
        </attribute>
        <attribute name="title">
         <string> 第二页</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset>
          <normaloff>:/images/page3.bmp</normaloff>:/images/page3.bmp</iconset>
        </attribute>
        <attribute name="title">
         <string>第三页</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actInit"/>
   <addaction name="m_actClear"/>
   <addaction name="m_actAppend"/>
   <addaction name="m_actInsert"/>
   <addaction name="m_actDelete"/>
  </widget>
  <action name="m_actInit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/init.bmp</normaloff>:/images/init.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </action>
  <action name="m_actClear">
   <property name="icon">
    <iconset>
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>全清空</string>
   </property>
   <property name="toolTip">
    <string>全清空</string>
   </property>
  </action>
  <action name="m_actAppend">
   <property name="icon">
    <iconset>
     <normaloff>:/images/append.bmp</normaloff>:/images/append.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="m_actInsert">
   <property name="icon">
    <iconset>
     <normaloff>:/images/insert.bmp</normaloff>:/images/insert.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="m_actDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/images/delete.bmp</normaloff>:/images/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除项</string>
   </property>
   <property name="toolTip">
    <string>删除项</string>
   </property>
  </action>
  <action name="m_actCheckAll">
   <property name="text">
    <string>全勾选</string>
   </property>
   <property name="toolTip">
    <string>全勾选</string>
   </property>
  </action>
  <action name="m_actCheckNone">
   <property name="text">
    <string>不勾选</string>
   </property>
   <property name="toolTip">
    <string>不勾选</string>
   </property>
  </action>
  <action name="m_actCheckInverse">
   <property name="text">
    <string>反勾选</string>
   </property>
   <property name="toolTip">
    <string>反勾选</string>
   </property>
  </action>
  <action name="m_actCheck">
   <property name="icon">
    <iconset>
     <normaloff>:/images/item.bmp</normaloff>:/images/item.bmp</iconset>
   </property>
   <property name="text">
    <string>勾选项</string>
   </property>
   <property name="toolTip">
    <string>勾选项</string>
   </property>
  </action>
  <action name="m_actExit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_actExit</sender>
   <signal>triggered()</signal>
   <receiver>ListWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actCheck</sender>
   <signal>triggered()</signal>
   <receiver>m_actCheckInverse</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
